includes/templates/story-templates/fandom-cta.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:amp/amp-story-page {"mediaType":"image","backgroundColors":"[{}]"} -->
 <amp-story-page style="background-color:#ffffff" id="150fd056-a21a-4273-962b-59e2ea7e778c" class="wp-block-amp-amp-story-page"><amp-story-grid-layer template="fill"><amp-img layout="fill" src="AMP_STORY_TEMPLATES_URL/fandom-cta.jpg" style="object-position:initial"></amp-img></amp-story-grid-layer><amp-story-grid-layer template="fill" style="opacity:1"></amp-story-grid-layer>
     <!-- wp:amp/amp-story-text {"tagName":"h2","align":"center","customTextColor":"#ffffff","autoFontSize":23,"positionTop":46,"positionLeft":8,"height":94,"width":275} -->
-    <amp-story-grid-layer data-block-name="amp/amp-story-text" template="vertical"><div class="amp-story-block-wrapper" id="8d2f7f12-1f26-4f71-a3d5-d20b7dd20ea0" style="position:absolute;top:46%;left:8%;width:83.84%;height:17%"><h2 style="color:#ffffff;text-align:center;display:flex" class="wp-block-amp-amp-story-text has-text-color"><amp-fit-text layout="flex-item" class="amp-text-content">Check your knowledge in Game of Thrones fan-made Trivia!</amp-fit-text></h2></div></amp-story-grid-layer>
+    <h2 style="color:#ffffff;text-align:center;display:flex" class="wp-block-amp-amp-story-text has-text-color"><amp-fit-text layout="flex-item" class="amp-text-content">Check your knowledge in Game of Thrones fan-made Trivia!</amp-fit-text></h2>
```

Comment:
One thing that gives me pause is that now the blocks are direct children of `amp-story-page`. If this were rendered straight out to the frontend this would be an AMP validation error because the elements need to be inside of `amp-story-grid-layer`. This is more of a theoretical concern because the sanitizers have to run on the markup anyway for it to be valid, so it's not a big deal.

Nevertheless, in the future when it is possible for there to be multiple `InnerBlocks` in a single block, then this would seem like a good future improvement to have two separate `InnerBlocks` for the `amp-story-page`: one `InnerBlocks` which is wrapped by `amp-story-grid-layer` to contain all the blocks like this, and another `InnerBlocks` which is wrapped by `amp-story-cta-layer`.

Cf. #1993.

Reply:
This probably won't be rendered straight out to the frontend at any moment, otherwise, any dynamic parts (e.g. potential dynamic blocks) wouldn't really display anything.

Worst case scenario we could add the wrappers back in the future and then create deprecation for that logic, however, fairly certain that this won't happen :)

> Nevertheless, in the future when it is possible for there to be multiple InnerBlocks in a single block, then this would seem like a good future improvement

Good point.